model/funque_float.json

```diff
@@ -13,20 +13,39 @@
         "model_type": "LIBSVMNUSVR",
         "feature_opts_dicts": [
             {
+                "enable_spatial_csf": false,
                 "enable_resize": true,
-                "vif_levels": 2
-            },
-            {},
-            {},
-            {},
-            {}
+
+                "spatial_csf_filter": 5,
+                "wavelet_csf_filter": 1,
+
+                "vif_levels": 4,
+                "adm_levels": 4,
+                "ssim_levels": 4,
+                "ms_ssim_levels": 4,
+                "strred_levels": 4
+            }
         ],
         "feature_names": [
-            "FUNQUE_feature_vif_scale0_score",
-            "FUNQUE_feature_vif_scale1_score",
-            "FUNQUE_feature_motion_score",
-            "FUNQUE_feature_adm2_score",
-            "FUNQUE_float_ssim"
+            "FUNQUE_feature_vif_score",
+            "FUNQUE_feature_vif_scale0_score", "FUNQUE_feature_vif_scale1_score",
+            "FUNQUE_feature_vif_scale2_score", "FUNQUE_feature_vif_scale3_score",
+
+            "FUNQUE_feature_adm_score",
+            "FUNQUE_feature_adm_scale0_score", "FUNQUE_feature_adm_scale1_score",
+            "FUNQUE_feature_adm_scale2_score", "FUNQUE_feature_adm_scale3_score",
+
+            "FUNQUE_feature_ssim_scale0_score", "FUNQUE_feature_ssim_scale1_score",
```

Comment:
it would be great if like `ms_ssim` these reported both mean and mink3 pooled scores.